preview_generator/preview/builder/image__inkscape.py

```diff
@@ -16,6 +15,29 @@
 from preview_generator.utils import ImgDims
 from preview_generator.utils import executable_is_available
 
+INKSCAPE_EXECUTABLE = "inkscape"
+DEFAULT_INKSCAPE_VERSION = "1.0"
+
+INKSCAPE_092_SVG_TO_PNG_OPTIONS = ["--export-area-drawing", "-e"]
+INKSCAPE_100_SVG_TO_PNG_OPTIONS = ["--export-area-drawing", "--export-type=png", "-o"]
+
+
+def get_inkscape_version() -> float:
+    return float(os.environ.get("INKSCAPE_VERSION", default=DEFAULT_INKSCAPE_VERSION))
```

Comment:
This value cannot change during execution so it can directly be put in a variable:
```suggestion
INKSCAPE_VERSION = float(os.environ.get("INKSCAPE_VERSION", default=DEFAULT_INKSCAPE_VERSION))
```

Reply:
By the way, why don't we use `inkscape --version`? (in which case would we set INSCAPE_VERSION to something different from the actual inkscape version?

Reply:
It does not seem right to ask users or sysadmin information that we have readily available. I think the version of Inkscape should be detected at startup time, or on first use (or on each use).

A mismatch is bound to happen, either because the user has forgot (or didn't know) to set this value, or because the user will update their system and don't think of updating this value.

This *will* break for someone somewhere until inkscape < 1.0 vanishes. This makes preview-generator and Tracim harder to install for no good reason.

The output of `inkscape --version` in older inkscape versions will never change because they are already released. I suggest defaulting to the behavior for inkscape 1.0, and try detecting Inkscape < 1.0 with something like the following and not put any unnecessary burden on users:

``python
subprocess.run(['inkscape', '--version'], stdout = subprocess.PIPE).stdout.startswith(b'Inkscape 0.')
```

Reply:
I made changes in this direction.

Reply:
looks good to me

Reply:
By the way we already call `inkscape --version` here, we might as well reuse it / store the version at this point?

https://github.com/algoo/preview-generator/pull/264/files#diff-ee294db389dbc519767c5577a1b472e5c4590eed58554277bdc860805fedc4b9R52